gobcore/typesystem/gob_geotypes.py

```diff
@@ -72,25 +74,35 @@ class Point(GEOType):
     sql_type = geoalchemy2.Geometry('POINT')
 
     @classmethod
-    def from_value(cls, value):
+    def from_value(cls, value, **kwargs):
         """Instantiates the GOBType Point, with either a database value, a geojson or WKT string"""
 
         if isinstance(value, geoalchemy2.Geometry):
             return cls(value.desc, srid=value.srid)
 
+        if isinstance(value, dict):
+            # serialize possible geojson
+            value = json.dumps(value)
+
         # if is geojson dump to wkt string
+        value = cls._wkt_from_geojson(kwargs, value)
+
+        # is wkt string
+        return cls(value)
+
+    @classmethod
+    def _wkt_from_geojson(cls, kwargs, value):
         try:
-            wkt_string = wkt.dumps(json.loads(value))
+            precision = kwargs['precision'] if 'precision' in kwargs else cls._precision
+            wkt_string = wkt.dumps(json.loads(value), decimals=precision)
             value = wkt_string
```

Comment:
Maybe assign the output of wkt.dumps directly to value.  Seems cleaner and more in sync with other functions